modules/json/test/src/smithy4s/http/json/FaceCard.scala

```diff
@@ -20,10 +20,12 @@ import smithy4s.schema.Schema._
 
 sealed abstract class FaceCard(_value: String, _name: String, _intValue: Int)
```

Comment:
there is already a generated `FaceCard` (from a smithy spec) in the example module, shouldn't we use that instead?

Reply:
Imo you can definitely update it to be the case. I think I duplicated it at the time because the PRs were in flight at roughly the same time.. don't remember to be honest 😅